src/components/atoms/UserAvatar/UserAvatar.scss

```diff
@@ -1,9 +1,15 @@
 @import "scss/constants";
 
 .UserAvatar {
```

Comment:
This component and it's styles are used in a lot of places and this change might break something. Please check every single usage and make sure it doesn't break anything visually. Chats, navbar, profile modals, etc...

Reply:
Checked usages, and generally doesn't break things. In the one place scaling is preferred i.e. the party map, it did show inconsistency, so I've made a `size` prop that will default to this behavior when not explicitly set otherwise. b40bc48846f23c0b8cebb8e9f7f5941bfc5b29bf

Reply:
same as https://github.com/sparkletown/sparkle/pull/1870/files#r674052990